services/impl/src/main/java/org/exoplatform/automatic/translation/listener/analytics/AutomaticTranslationListener.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2021 eXo Platform SAS.
```

Comment:
```suggestion
 * Copyright (C) 2022 eXo Platform SAS.
```

Reply:
Those files has been moved as is from Analytics in fact, no need to change the copyright year IMO
https://github.com/exoplatform/analytics/blob/36a57fdc1d911be8e2bfbb2665d040873f4765dc/analytics-listeners/src/main/java/org/exoplatform/analytics/listener/translation/AutomaticTranslationListener.java#L2

Reply:
changed anyway